plugin/entryBase.go

```diff
@@ -128,6 +133,15 @@ func (e *EntryBase) SetAttributes(attr EntryAttributes) *EntryBase {
 	return e
 }
 
+// Prefetched tags an entry as having prefetched data that is populated
```

Comment:
How about something like:

"Prefetched marks the entry as a prefetched entry. A prefetched entry is an entry that was fetched as part of a batch operation that fetched multiple levels of hierarchy at once. Volume directories and files are good examples of prefetched entries (see the volume package for more details)."

Reason for the reword is b/c I don't think it's worth including the "use this if you want your entry's data to be updated" -- the updating/refreshing is a part of a prefetched entry's semantics.

Reply:
Plus, it's also easier to tell people "If you fetched this thing via a batched operation that fetched multiple levels of hierarchy, then use this helper to mark it as prefetched." Maybe that sentence alone is good enough.

Reply:
Kind of wondering if `Prefetch` is a good name here. By definition, prefetch means you're getting data you'll know you need soon, but that you don't know for sure you'll need. In our case, that's not true for the first level in the hierarchy since the prefetch was trigged by a list call to the parent (meaning that we know for sure we'll need the first level, instead of "we might need the first level"). That's why I'm wondering if something like `BulkFetched`/`BatchFetched` might be better? It would be consistent with how we're thinking about this stuff and how we intend for it to be used (e.g. the external plugin need arose b/c a user wanted a way to pass-in batch-fetched data to Wash).

I could also be talking nonsense here so if I am, please feel free to ignore this comment.

Reply:
I think the name is fine. If someone is confused about the documentation, we'll reconsider.